app/services/consult_service.py

```
@@ -0,0 +1,330 @@
+
+        # 6. 스트림 완료 후, 전체 응답을 DB에 저장
+        agent_log = ConsultLog(
+            user_id=user_id,
+            session_id=session_id,
+            role=ConsultRoleEnum.AGENT,
+            content=full_response_text  # 취합된 최종 텍스트 저장
+        )
+        db.add(agent_log)
+        db.commit()
+
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)